.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# Order is important. The last matching pattern has the most precedence.
```

Comment:
I would say `Order of the rules` to make it clear that order of the reviewer is ignored.

Reply:
Addressed in 6802770.